### PR TITLE
Fix gh-860 // Update FAQ

### PR DESCRIPTION
**Pull request by St19Galla**
Updated the Scratch 2 Open Source info as per @kaschm and @thisandagain in #860 :)

### CODE REVIEW

**Comment by mewtaylor** [comment]
@St19Galla looks good to me – will let @kaschm give it a look too

**Comment by kaschm** [comment]
:+1: thanks, @St19Galla !

File: src/views/faq/l10n.json [hunk 1]
```diff
@@ -86,7 +86,7 @@
...
     "faq.sourceCodeTitle":"Where can I find the source code for Scratch?",
-    "faq.sourceCodeBody":"The source code for Scratch 2 (website and editor) is not yet available. The source code for <a href =\"/scratch_1.4\">Scratch 1.4</a>, written in Squeak, is available <a href=\"https://github.com/LLK/Scratch_1.4\">on GitHub</a>. The source code for the first version of the Scratch website - ScratchR - is available <a href=\"http://svn.assembla.com/svn/scratchr/\">on Assembla</a>.",
+    "faq.sourceCodeBody":"The source code for the Scratch 2 editor can be found <a href=\"https://github.com/LLK/scratch-flash\">on GitHub</a>. The source code for <a href =\"/scratch_1.4\">Scratch 1.4</a>, written in Squeak, is also available <a href=\"https://github.com/LLK/Scratch_1.4\">on GitHub</a>. For updated information on development projects relating to the Scratch website, please visit our <a href=\"/developers\">Developer Page</a>.",
```
Comment (mewtaylor) [inline src/views/faq/l10n.json]: small tweak if you have a moment - in both of the Github links, could we take the word "on" out of the anchor and have it appear beforehand?

**Comment by St19Galla** [comment]
@mewtaylor done! :)

**Comment by chrisgarrity** [comment]
QA: verified updated text and links

**Pull request merged**